### PR TITLE
feat(project): add validation support

### PR DESCRIPTION
**Pull request by RWOverdijk**

### CODE REVIEW

File: src/component/view/bootstrap/form-group.html [hunk 1]
```diff
@@ -11,7 +11,14 @@
           classes="${is('horizontal', 'col-sm-2', behavior)} control-label"
         >
           <slot name="element">
-            <form-element slot="before" if.bind="element" element.bind="element" value.bind="value"></form-element>
+            <form-element
+              slot="before"
+              name.bind="name"
+              if.bind="element"
```
Comment (jeremyvergnas) [inline src/component/view/bootstrap/form-group.html:17]: use `show.bind` instead?

File: src/component/aurelia-form.js [hunk 2]
```diff
@@ -1,28 +1,107 @@
...
+    // Specific configured, something else triggered. Return.
+    if (setTrigger < triggers.changeOrBlur && triggers[trigger] !== setTrigger) {
+      return;
+    }
+
+    this.validate(event.target.name);
+  }
+
+  validate(property) {
+    return this.validationController.validate({object: this.entity, propertyName: property});
```
Comment (jeremyvergnas) [inline src/component/aurelia-form.js:79]: What happen if no `property` argument is given? It will just validate the whole form?

**Review by jeremyvergnas — COMMENTED** [review]
Should use `show.bind` instead of `if.bind`

File: src/component/aurelia-form.js (hunk 2 above)
Comment (RWOverdijk) [reply src/component/aurelia-form.js:79]: Jup

File: src/component/view/bootstrap/form-group.html (hunk 1 above)
Comment (RWOverdijk) [reply src/component/view/bootstrap/form-group.html:17]: @jeremyvergnas This doesn't contain a slot that will be re-rendered.

**Pull request merged**